.ci/docker/Dockerfile

```diff
@@ -121,7 +121,7 @@ RUN curl "https://downloads.haskell.org/~ghcup/$ghcup_version/x86_64-linux-ghcup
  && ghcup install ghc $ghc_version --set \
  && ghcup install cabal $cabal_version --set
 
-ARG UBUNTU_VERSION=focal-20210416
+ARG UBUNTU_VERSION=jammy-20240125
 FROM ubuntu:$UBUNTU_VERSION AS run
```

Comment:
To prevent future mismatches like this, consider removing the value in both places in the docker file, just have two times:
```
ARG UBUNTU_VERSION
```

And then setting the version in one place, in `.ci/docker/build-and-publish-docker-image.sh` with a `--build-arg`

Reply:
You're quite right.